ethereum/contracts/GovernanceStructs.sol

```diff
@@ -52,6 +52,14 @@ contract GovernanceStructs {
         bytes32 recipient;
     }
 
+    struct RecoverChainId {
+        bytes32 module;
+        uint8 action;
```

Comment:
maybe this ship has sailed, but I don't think `module` and `action` should be in these structs at all, it's something the parser could check and then throw away